Dockerfile

```diff
@@ -4,51 +4,5 @@ ARG RUST_TOOLCHAIN="1.67.1"
 # Adding rust binaries to PATH.
 ENV PATH="$PATH:/root/.cargo/bin"
 
-# Install all required packages in one go to optimize the image
-# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run
-# DEBIAN_FRONTEND is set for tzdata.
-RUN apt-get update && \
-    DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -y \
-    curl gcc git python3 python3-pip shellcheck \
-    # kcov dependencies
-    libssl-dev tzdata cmake g++ pkg-config jq libcurl4-openssl-dev libelf-dev \
-    libdw-dev binutils-dev libiberty-dev make \
-    # utilities to build kernels
-    cpio bc flex bison wget xz-utils fakeroot \
-    # utilities to build usersapce packages
-    autoconf autoconf-archive automake libtool \
-    # bindgen dependency
-    libclang-dev \
```

Comment:
@andreeaflorescu This is required to run bindgen for libgpiod crate, which is done while buidling vhost-device crate. This commit broke vhost-device:

https://github.com/rust-vmm/vhost-device/pull/308